rust/processor/migrations/2023-09-08-161603_add_ans_v2/up.sql

```diff
@@ -0,0 +1,78 @@
+-- Tracks ans v1 and v2 records
+CREATE TABLE IF NOT EXISTS current_ans_lookup_v2 (
+  domain VARCHAR(64) NOT NULL,
+  -- if subdomain is null set to empty string
+  subdomain VARCHAR(64) NOT NULL,
+  token_standard VARCHAR(10) NOT NULL,
+  token_name VARCHAR(140),
+  registered_address VARCHAR(66),
+  expiration_timestamp TIMESTAMP NOT NULL,
+  last_transaction_version BIGINT NOT NULL,
+  is_deleted BOOLEAN NOT NULL,
+  inserted_at TIMESTAMP NOT NULL DEFAULT NOW(),
+  -- Constraints
+  PRIMARY KEY (domain, subdomain, token_standard)
+);
+CREATE INDEX IF NOT EXISTS ans_v2_ts_index on current_ans_lookup_v2 (token_standard);
```

Comment:
why? Let's not add redundant indices, especially if this one has only a cardinality of 2.